library/Icinga/Web/Response.php

```diff
@@ -198,6 +205,30 @@ public function getRequest()
         return $this->request;
     }
 
+
+    /**
+     * Get whether to instruct client side script code to reload JS
+     *
+     * @return bool
+     */
+    public function isReloadJs()
+    {
+        return $this->reloadJs;
+    }
+
+    /**
+     * Set whether to instruct client side script code to reload JS
+     *
+     * @param   bool    $reloadJs
+     *
+     * @return  $this
+     */
+    public function setReloadJs($reloadJs)
+    {
+        $this->reloadJs = $reloadJs;
+        return $this;
```

Comment:
Add newline before return.

Reply:
In Response.php there are other cases like this. Should i add newlines too?